1_crosswalk_fetch/src/fetch_crosswalk.R

```diff
@@ -209,13 +209,14 @@ fetch_micorps_sites <- function(ind_file) {
 }
 
 #' use `dummy` to trigger rebuilds. I am using the date, as a light reminder of when it was changed
-fetch_wqp_lake_sites <- function(ind_file, characteristicName, dummy){
+fetch_wqp_lake_sites <- function(ind_file, characteristicName, bBox, dummy){
 
   message('warning, avoiding geojson output due to issue with services 2020-01-19
           hacking resultCount as 200')
 
   # when switching back in the future, seems lat and lon were the names for LatitudeMeasure & LongitudeMeasure
-  lake_sites_sf <- whatWQPsites(siteType = "Lake, Reservoir, Impoundment", characteristicName = characteristicName) %>%
+  lake_sites_sf <- whatWQPsites(siteType = "Lake, Reservoir, Impoundment", characteristicName = characteristicName,
+                                bBox = bBox) %>%
```

Comment:
Adding the bounding box was necessary since the full national pull (no bbox or other geo filter specified) was consistently erroring out.